redbeams/src/main/java/com/sequenceiq/redbeams/converter/stack/AllocateDatabaseServerV4RequestToDBStackConverter.java

```diff
@@ -160,9 +170,19 @@ public DBStack convert(AllocateDatabaseServerV4Request source, String ownerCrnSt
         dbStack.setDBStackStatus(new DBStackStatus(dbStack, DetailedDBStackStatus.PROVISION_REQUESTED, now.toEpochMilli()));
         dbStack.setResourceCrn(crnService.createCrn(dbStack));
         dbStack.setTags(getTags(dbStack, source.getClusterCrn(), environment));
+        dbStack.setSslConfig(getSslConfig(source, dbStack));
         return dbStack;
     }
 
+    public SslConfig getSslConfig(AllocateDatabaseServerV4Request source, DBStack dbStack) {
+        SslConfig sslConfig = new SslConfig();
+        if (sslEnabled && source.getSslConfiguration() != null && SslMode.isEnabled(source.getSslConfiguration().getMode())) {
+            sslConfig.setSslCertificates(databaseServerSSlCertificateConfig.getCertsByPlatform(dbStack.getCloudPlatform()));
```

Comment:
So we are storing the whole certificate text in the DB? Would not it be better to just store some internal cert ID for `CLOUD_PROVIDER_OWNED` certs to save storage space?

Reply:
we do the same in CB codebase as well